test/integration/cx_limit_integration_test.cc

```diff
@@ -37,6 +37,24 @@ class ConnectionLimitIntegrationTest : public testing::TestWithParam<Network::Ad
 
   void initialize() override { BaseIntegrationTest::initialize(); }
 
+  AssertionResult waitForConnections(uint32_t envoy_downstream_connections) {
+    // The multiplier of 2 is because both Envoy's downstream connections and
+    // the test server's downstream connections are counted by the global
+    // counter.
+    uint32_t expected_connections = envoy_downstream_connections * 2;
+
+    for (int i = 0; i < 10; ++i) {
+      if (Network::AcceptedSocketImpl::acceptedSocketCount() == expected_connections) {
+        return AssertionSuccess();
+      }
+      timeSystem().advanceTimeWait(std::chrono::milliseconds(500));
```

Comment:
CI shows a build failure because advanceTimeWait does not exist.  Seems that before PR 10551 you need something like:
timeSystem().sleep(std::chrono::milliseconds(500));